pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1628,17 +1628,24 @@ Question.Boolean:
       - answers
       - trueContent
       - falseContent
+      - renderMode
     properties:
       questionType:
         type: string
         default: BOOLEAN
         enum:
           - BOOLEAN
-      trueContent:
+      renderMode:
         type: string
+        default: RADIO_BUTTONS
+        enum:
+          - CHECKBOX
+          - RADIO_BUTTONS
+      trueContent:
+        type: string | null (for CHECKBOX renderMode)
```

Comment:
Should it be an empty string in this case (for CHECKBOX mode) ?

Reply:
Or should I use `nullable` like below ?
![image](https://user-images.githubusercontent.com/7396837/161296799-828f9a52-874b-4f88-8320-5ea4b6dedc3c.png)

Reply:
Making those fields `nullable` is a breaking API change. How is the question supposed to appear? I was under the impression that this was supposed to mirror the `renderMode` of a `PICKLIST` question. Radio-like behavior using a checkbox appearance. This is sounding like the behavior is changing to be more in-line with the `AGREEMENT` question type.

Reply:
Since DDP-7707 doesn't have any acceptance criteria attached to it. My understanding of the behavior described in DDP-7706 does seems to agree with the latter situation. In that case, both the `trueContent` and `falseContent` properties should have the values for their `nullable` properties set to `true` (like in the example you quoted above)

Reply:
> If the front-end is just going to ignore these values

Right, for **CHECKBOX** mode the **_trueContent_**/**_falseContent_** values will be ignored.

Fixed in e7f3453f7f955032b8df5ff6d71206140ab50d01